### PR TITLE
Add metadata for doc ocr cpu/gpu 22.10.2 patch

### PR DESCRIPTION
**Pull request by irinel-birsan-uipath**

### CODE REVIEW

File: metadata/UiPathDocumentOCR_CPU__4__metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	"stagingUri": "[staging-uri]",
+	"projectName": "UiPath Document Understanding",
+	"projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+	"tenantName": "UiPath",
+	"minAIFabricVersion": "22.10",
+	"languageVersion": 4,
+	"version": 4,
+	"customVersion": "22.10.2",
+	"imagePath": "du-doc-ocr-cpu:22.10.2-sf-rc1",
+	"parametersFileJSON": "{\"params\": [],\"probes\": [{\"probe\": \"startup\",\"initialDelaySeconds\": \"30\",\"periodSeconds\": \"10\",\"timeoutSeconds\": \"5\",\"successThreshold\": \"1\",\"failureThreshold\": \"60\"},{\"probe\": \"readiness\",\"initialDelaySeconds\": \"30\",\"periodSeconds\": \"15\",\"timeoutSeconds\": \"45\",\"successThreshold\": \"1\",\"failureThreshold\": \"2\"},{\"probe\": \"liveness\",\"initialDelaySeconds\": \"30\",\"periodSeconds\": \"60\",\"timeoutSeconds\": \"45\",\"successThreshold\": \"1\",\"failureThreshold\": \"3\"}]}"
```
Comment (vivekjhauipath) [inline metadata/UiPathDocumentOCR_CPU__4__metadata.json:25]: @irinel-birsan-uipath would these values be enough for CPU?

**Review by vivekjhauipath — APPROVED** [review]
LGTM except for probe settings, which should be validated

File: metadata/UiPathDocumentOCR_CPU__4__metadata.json (hunk 1 above)
Comment (irinel-birsan-uipath) [reply metadata/UiPathDocumentOCR_CPU__4__metadata.json:25]: We don't have an issue reported on this. I have copied over from the previously released version.

**Review by bogdan-dumitru-uipath — COMMENTED** [review]
LGTM

**Pull request merged**